src/pages/registration/SupportModalContent.tsx

```diff
@@ -22,11 +29,99 @@ export const SupportModalContent = ({ registration }: SupportModalContentProps)
+        closeModal();
+      },
+    }
+  );
+
+  const createSupportTicketMutation = useMutation(
+    async (message: string) => await createTicket(registration.id, 'GeneralSupportCase', true),
+    {
+      onSuccess: (data, message) => addMessageMutation.mutate({ ticketId: data.data?.id || '', message: message }),
+      onError: () => dispatch(setNotification({ message: t('feedback.error.send_message'), variant: 'error' })),
+    }
+  );
+  const isLoading = ticketsQuery.isLoading || addMessageMutation.isLoading || createSupportTicketMutation.isLoading;
+
+  let tickets = ticketsQuery.data?.tickets ?? [];
+  tickets = tickets.filter((ticket) => ticket.type === 'GeneralSupportCase');
+  const currentSupportTicket = tickets.length > 0 ? tickets.at(tickets.length - 1) : null;
+
+  const renderActiveTicketPresent = () => {
+    return <Typography>{t('registration.support.curator_help.already_in_progress')}</Typography>;
+  };
+
+  const renderLoadingTickets = () => {
+    return <Skeleton variant="text" sx={{ fontSize: '10rem' }} />;
+  };
+
+  const renderCreateTicket = () => {
+    return (
+      <>
+        <Typography variant={'h3'} marginBottom={2} gutterBottom>
+          {t('registration.support.curator_help.header')}
+        </Typography>
+        <Trans t={t} i18nKey="registration.support.curator_help.description" components={[<Typography paragraph />]} />
+        <MessageForm
+          confirmAction={async (message) => {
+            if (message) {
+              createSupportTicketMutation.mutate(message);
+            }
+          }}
+        />
+      </>
+    );
+  };
+
   return (
-    <ActionPanelContent
-      tickets={ticketsQuery.data?.tickets ?? []}
-      refetchData={() => ticketsQuery.refetch()}
-      registration={registration}
-    />
+    <>
+      <Box>
+        <Grid container spacing={3}>
+          <Grid item md={6} sm={12}>
+            <Typography variant={'h3'} marginBottom={2} gutterBottom>
+              {t('registration.support.self_help.header')}
+            </Typography>
+            <MuiLink
+              sx={{
+                fontWeight: 'bold',
+                color: 'primary.main',
+                display: 'flex',
+                gap: '0.5rem',
+                mb: '1rem',
+              }}
+              target="_blank"
+              rel="noopener noreferrer"
+              href={'https://sikt.no/tjenester/nasjonalt-vitenarkiv-nva/hjelpeside-nva'}>
+              <Box bgcolor={'white'} padding={1} marginBottom={2} width={'100%'} sx={{ borderRadius: 2 }}>
+                <Grid container>
+                  <Grid item xs={11}>
+                    <Typography variant={'body1'}>{t('footer.help_page')}</Typography>
+                  </Grid>
+                  <Grid item xs={1}>
+                    <OpenInNew fontSize="small" />
+                  </Grid>
+                </Grid>
+              </Box>
+            </MuiLink>
```

Comment:
Ser ut som det er en del overflødig kode og tags her også, som gjør det vanskelig å se hvor ting styles. Ser ut som det som ønsker er bare en Link (`<a>`) med spesiell styling
```suggestion
            <MuiLink
              sx={{
                display: 'flex',
                justifyContent: 'space-between',
                gap: '0.5rem',
                mb: '1rem',
                bgcolor: 'white',
                padding: '0.5rem',
                width: '100%',
                borderRadius: '0.5rem',
              }}
              target="_blank"
              rel="noopener noreferrer"
              href={'https://sikt.no/tjenester/nasjonalt-vitenarkiv-nva/hjelpeside-nva'}>
              <Typography>{t('footer.help_page')}</Typography>
              <OpenInNew />
            </MuiLink>
```

Reply:
Bra.